CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+## v0.6.1 - 2021-02-09
+
+This is a patch release which includes mainly bug fixes.
+
+> **NOTE**: Please read the updating guidelines [here](https://github.com/kinvolk/lokomotive/blob/v0.6.1/CHANGELOG.md#updating-from-v060).
+
+### Changes in v0.6.1
+
+#### Development
+
+- Velero: Add tolerations to Restic plugin ([#1348](https://github.com/kinvolk/lokomotive/pull/1348)).
+- Velero: Add e2e tests ([#1353](https://github.com/kinvolk/lokomotive/pull/1353)).
+- Update all Go dependencies ([#1358](https://github.com/kinvolk/lokomotive/pull/1358)).
+
+#### Terraform Provider Updates
+- Update Packet (Equinux Metal) Terraform provider to 3.2.1 that fixes the provisioning failures of
+  `n2.xlarge.x86` machines ([#1349](https://github.com/kinvolk/lokomotive/pull/1349)).
+
+#### Bug fixes
+
+- Prefix `ETCD_` for standard etcd environment variables only ([#1308](https://github.com/kinvolk/lokomotive/pull/1308)).
+- Update Restic TolerationSeconds type to integer and add conditional checks ([#1365](https://github.com/kinvolk/lokomotive/pull/1365)).
+
+#### Docs
+- Add missing `provider` parameter ([#1354](https://github.com/kinvolk/lokomotive/pull/1354)).
+- Update RELEASING document to add steps to update the documentation website entry ([#1326](https://github.com/kinvolk/lokomotive/pull/1326)).
+- Improvements to the Lokomotive release process documentation ([#1341](https://github.com/kinvolk/lokomotive/pull/1341)).
+
+### Updating from v0.6.0
+
+#### Cluster update steps
+
+> **NOTE:** Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a
+> version older than `v0.6.0`, update to `v0.6.0` first and only then proceed with the update to `v0.6.1`.
+
+Please perform the following manual steps in your cluster configuration directory.
+
+1. Download the release bundle.
+
+  ```bash
+  curl -LO https://github.com/kinvolk/lokomotive/archive/v0.6.1.tar.gz
+  tar -xvzf v0.6.1.tar.gz
```

Comment:
Also shouldn't we add steps for building lokoctl? Otherwise, it will use the old binary right?

After the download step, we should point to the installer guide I guess.

Reply:
Good catch. Pointed to the installation guide as the first step.